docs/source/markdown/podman-create.1.md

```diff
@@ -646,7 +646,7 @@ Valid _mode_ values are:
 - **ns:**_path_: path to a network namespace to join;
 - **private**: create a new namespace for the container (default)
 - **slirp4netns[:OPTIONS,...]**: use **slirp4netns**(1) to create a user network stack. This is the default for rootless containers. It is possible to specify these additional options:
-  - **allow_host_loopback=true|false**: Allow the slirp4netns to reach the host loopback IP (`10.0.2.2`). Default is false.
+  - **allow_host_loopback=true|false**: Allow the slirp4netns to reach the host loopback IP (`10.0.2.2` or `host.containers.internal`). Default is false.
```

Comment:
Change also needs to be made in `podman-run.1.md` in the same directory. Also, it would be clearer if this said "10.0.2.2, which is added to `/etc/hosts` as `host.containers.internal`"